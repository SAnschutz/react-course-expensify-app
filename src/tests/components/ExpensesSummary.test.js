import React from 'react';
import { shallow } from 'enzyme';
import { ExpensesSummary } from '../../components/ExpensesSummary';



test('should render ExpensesSummary correctly with one expense', () => {
    const wrapper = shallow(<ExpensesSummary expenseCount={1} expensesTotal={33}/>);
    expect(wrapper).toMatchSnapshot();

});

test('should render ExpensesSummary correctly with multiple expenses', () => {
    const wrapper = shallow(<ExpensesSummary expenseCount={342} expensesTotal={3423423432}/>);
    expect(wrapper).toMatchSnapshot();
});