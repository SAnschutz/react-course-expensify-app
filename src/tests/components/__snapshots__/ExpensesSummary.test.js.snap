// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render ExpensesSummary correctly with multiple expenses 1`] = `
<div>
  <p>
    Viewing 
    342
     
    expenses
     totalling 
    $34,234,234.32
  </p>
</div>
`;

exports[`should render ExpensesSummary correctly with one expense 1`] = `
<div>
  <p>
    Viewing 
    1
     
    expense
     totalling 
    $0.33
  </p>
</div>
`;
